test/fixtures/output/kafka/link/describe-link-with-tasks.golden

```diff
@@ -0,0 +1,16 @@
++---------------------+-----------+
```

Comment:
This style of output is something we've been moving away from for the past two years. Can we break this out into two separate commands?

Reply:
I understand the concern. I've asked for my team's input. 

A downside of creating a new command is obviously its a new command for the customer to learn. Let me think about it. 

How bad is it to have these two tables?

Reply:
> How bad is it to have these two tables?

It's not what users expect, it makes it hard to standardize across commands, and results in some ambiguity when it comes to formatting the serialized output.

Reply:
I get that @brianstrauch.

I'm going to get input from our PM @lukeknep here.

Reply:
@brianstrauch I've created a new command: `confluent kafka link describe-tasks`. Let me know your thoughts.

Reply:
Same comment here, let's rename this to `confluent kafka link tasks describe`